cloud-aws-cloudformation/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/connector/resource/upgrade/operation/AwsRdsUpgradeOperations.java

```diff
@@ -59,8 +60,10 @@ public void upgradeRds(AmazonRdsClient rdsClient, RdsEngineVersion targetVersion
                 .withDBInstanceIdentifier(dbInstanceIdentifier)
                 .withEngineVersion(targetVersion.getVersion())
                 .withAllowMajorVersionUpgrade(true)
-                .withApplyImmediately(true)
-                .withDBParameterGroupName(dbParameterGroupName);
+                .withApplyImmediately(true);
+        if (StringUtils.isNotEmpty(dbParameterGroupName)) {
+            modifyDBInstanceRequest.withDBParameterGroupName(dbParameterGroupName);
+        }
```

Comment:
Is this check necessary? wouldn't not adding the parameter group name to request would be the same as adding it but with null value?

Reply:
It can be null or empty string, and I didn't test it with empty string, decided to use the default value in the request.